### PR TITLE
fixing-title-in-osx

### PR DESCRIPTION
**Pull request by tesonep**
Fixing the title option in OSX.
Now it handled it as a file, showing finder if click on it.

### CODE REVIEW

File: src/OSWindow-Core/OSWorldRenderer.class.st [hunk 1]
```diff
@@ -214,6 +217,21 @@ OSWorldRenderer >> updateDamage: allDamage [
...
+{ #category : #activation }
+OSWorldRenderer >> updateWindowTitle [
+
+	osWindow title: Smalltalk imageFile fullName.
+	OSPlatform current sdlPlatform
+		afterSetWindowTitle: Smalltalk imageFile fullName
+		onWindow: osWindow
+]
+
+{ #category : #'as yet unclassified' }
```
Comment (guillep) [inline src/OSWindow-Core/OSWorldRenderer.class.st]: unclassified

File: src/OSWindow-SDL2/SDLNullPlatform.class.st [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+I am the default platform. All my methods are empty or have a default implementation
+"
+Class {
+	#name : #SDLNullPlatform,
+	#superclass : #SDLAbstractPlatform,
+	#category : #'OSWindow-SDL2-Bindings'
+}
+
+{ #category : #operations }
+SDLNullPlatform >> afterSetWindowTitle: aString onWindow: aOSSDLWindow [
```
Comment (guillep) [inline src/OSWindow-SDL2/SDLNullPlatform.class.st:11]: Why is it called `after*` ?

**Review by guillep — CHANGES_REQUESTED** [review]
Check my comments

File: src/OSWindow-SDL2/SDLNullPlatform.class.st (hunk 2 above)
Comment (tesonep) [reply src/OSWindow-SDL2/SDLNullPlatform.class.st:11]: Because it is invoked after changing the window title in SDL.

**Comment by MarcusDenker** [comment]
Blog post: https://thepharo.dev/2020/11/25/a-vm-bug-no-an-image-one/

(just for reference)

<blockquote><div> The Pharo Dev: Mastering Objects</div><div><strong><a href="https://thepharo.dev/2020/11/25/a-vm-bug-no-an-image-one/">A VM bug?… No, an image one</a></strong></div><div>Today, some Pharo users asked why we have lost a nice feature in Mac OS X. In this operating system, it is possible to Cmd-Click on the title of a window, if the window represents an open document,…</div></blockquote>

**Pull request merged**